dev/ProjectReunion_DLL/ProjectReunion_DLL.vcxproj

```diff
@@ -167,6 +168,36 @@
     <LinkIncremental>false</LinkIncremental>
     <TargetName>Microsoft.ProjectReunion</TargetName>
   </PropertyGroup>
+
+  <ItemDefinitionGroup Condition="'$(Configuration)'=='Debug'">
```

Comment:
consider gathering all this logic into a root directory.build.props file to ensure/enforce that all current and future reunion projects use hybrid CRT linkage.

Reply:
I'll do that in a future PR. I didn't modify the test projects, or even verify if they would just pick this up or need updates.

Will refactor in a follow up PR.